Development/nmos/settings.cpp

```diff
@@ -189,6 +201,28 @@ namespace nmos
                 return web::http::experimental::hsts{ (uint32_t)nmos::experimental::fields::hsts_max_age(settings), nmos::experimental::fields::hsts_include_sub_domains(settings) };
             return bst::nullopt;
         }
+
+        // Get server IP address of the interface to bind server connections
+        utility::string_t get_server_address(const settings& settings)
+        {
+            const auto server_address = nmos::experimental::fields::server_address(settings);
+            if (!server_address.empty())
+            {
+                // verify the server address match one of the interface addresses
```

Comment:
Why is this verification step necessary? If the app gave us a bogus address, it must be better to report that rather than silently ignore it. But I'd probably just trust the app and use `nmos::experimental::fields::server_address(settings)` directly...